### PR TITLE
Create Team Page and Nav Tabs Now Links to the Actual Page

### PR DESCRIPTION
**Pull request by jerry70450**
fixes issue #40 

Clicking the navigation tab on the home page will now actually take the user to that page. To keep the highlighting effect, a prop `value={index}` is passed into the `<Base>` component. The `index` is the index of the tab you wish to highlight in the `<Nav>` component. For the actual redirect, a `href={link}` prop is added to the `<Tab>` component.

### CODE REVIEW

**Comment by Sprit-Fun** [comment]
Adding the redirection of "Our Team" that redirects to localhost:3000/team

**Comment by jerry70450** [comment]
> Adding the redirection of "Our Team" that redirects to localhost:3000/team

@zguo123 is `/team` going to be the route for the "Our Team" page? Should we add it now or wait until that page is implemented? I don't believe that page exists yet.

**Comment by zguo123** [comment]
> > Adding the redirection of "Our Team" that redirects to localhost:3000/team
> 
> @zguo123 is `/team` going to be the route for the "Our Team" page? Should we add it now or wait until that page is implemented? I don't believe that page exists yet.

@jerry70450 Create it for now, leave it blank.

**Comment by Sprit-Fun** [comment]
Feature added. Waiting for review.

**Comment by jerry70450** [comment]
> > > Adding the redirection of "Our Team" that redirects to localhost:3000/team
> > 
> > 
> > @zguo123 is `/team` going to be the route for the "Our Team" page? Should we add it now or wait until that page is implemented? I don't believe that page exists yet.
> 
> @jerry70450 Create it for now, leave it blank.

Made the new page. Looks like this: 

image

**Review by Sprit-Fun — APPROVED** [review]
Feature added. Waiting for approval.

File: testing/main/src/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import {
   NotFound,
   StudentSurveys,
   ImportantLinks,
+  Team,
```
Comment (zguo123) [inline testing/main/src/App.js:9]: Good job!

File: testing/main/src/App.js [hunk 2]
```diff
@@ -37,7 +38,7 @@ function App() {
           <Route exact path="/" component={Home} />
           <Route exact path="/login" component={Login} />
           <Route exact path="/links" component={ImportantLinks} />
-
+          <Route exact path="/team" component={Team} />
           <Route exact path="/register" component={Registration} />
           <PrivateRoute exact path="/app" component={StudentSurveys} />
           <PrivateRoute exact path="/survey/detail" component={SurveyDetail} />
```
Comment (zguo123) [inline testing/main/src/App.js:44]: Excellent work!

**Review by zguo123 — APPROVED** [review]

**Pull request merged**